make/config/libmxnet.sym

```diff
@@ -12,4 +12,4 @@ Java_org_apache_mxnet*
 *Storage*Get*
 *on_enter_api*
 *on_exit_api*
-*MXAPISetLastError*
+*MXAPIHandleException*
```

Comment:
Isn't this covered by the first line? If not, should we expand the pattern for the first line instead?

Reply:
I believe the first line misses the prefix before MX. Not sure if we want to whitelist all *MX* APIs.

Reply:
Let's fix the first line then

Reply:
Updated. Please review again. Thanks!